cheatsheets/Secure_Product_Design_Cheat_sheet.md

```diff
@@ -0,0 +1,92 @@
+# Secure Product Design Cheat Sheet
+
+## Introduction
+
+The purpose of Secure Product Design is to ensure that all products meet or exceed the security requirements laid down by the organisation as part of the development lifecycle and to ensure that all security decisions made about the product being developed are explicit choices and result in the correct level of security for the product being developed.
+
+## Methodology
+
+As a basic start, establish secure defaults, minimise the attack surface area, and fail securely to those well-defined and understood, defaults.
```

Comment:
```suggestion
As a basic start, establish secure defaults, minimise the attack surface area, and fail securely to those well-defined and understood defaults.
```
Extraneous comma

Reply:
Strictly speaking, monolithic architectures have a smaller "attack surface" than micro-services architecture. Think of the analogy of the perimeter of a fully assembled jigsaw puzzle vs the total perimeter of all the disassembled pieces. That's one way of viewing the differences in the attack surface between those to architectural extremes. However, just because a monolithic architectures have a smaller "attack surface", that doesn't mean it should outweigh all other architectural concerns. (Perhaps that is stated elsewhere in this CS.) Re-usability, scalability, maintainability, complexity, etc. also need to be considered. Maybe its a given that that's understood. Or not. I personally don't like implicit assumptions, so I would suggest perhaps mentioning the overall balance. (Hint: Threat modeling is your friend.)